app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -60,7 +60,13 @@ def perform(_now)
     get_usps_proofing_results_job_request_delay_milliseconds / MILLISECONDS_PER_SECOND
 
   def proofer
-    @proofer ||= UspsInPersonProofing::Proofer.new
+    @proofer ||= begin
+      if IdentityConfig.store.usps_mock_fallback
+        UspsInPersonProofing::Mock::Proofer.new
+      else
+        UspsInPersonProofing::Proofer.new
+      end
+    end
```

Comment:
You could use [EnrollmentHelper#usps_proofer](https://github.com/18F/identity-idp/blob/fa47ef838fc7dd0860e4d56fd5517e8e6685328d/app/services/usps_in_person_proofing/enrollment_helper.rb#L86-L92) here

Reply:
Good idea, done.